app/services/integrations/aggregator/taxes/base_service.rb

```diff
@@ -20,7 +20,7 @@ def integration
         def integration_customer
           @integration_customer ||= begin
             int_customers = customer.integration_customers
-            int_customers.find { |ic| %w[IntegrationCustomers::AnrokCustomer IntegrationCustomers::AvalaraCustomer].include?(ic.type) }
+            int_customers.find { |ic| ic.tax_integration? }
```

Comment:
That could be even shorter if you like.

```suggestion
            int_customers.find(&:tax_integration?)
```